src/components/icons/Chevron/Chevron.tsx

```diff
@@ -16,14 +16,17 @@ export interface IChevronProps extends React.SVGProps<SVGSVGElement> {
   direction?: ChevronDirection;
 }
 
-const getRotation = (direction: ChevronDirection = 'down'): string => {
-  const directionMap = {
+const getRotation = (direction: ChevronDirection | number = 'down'): string => {
+  const directionMap: { [k in ChevronDirection]: '0' | '90' | '-90' | '180' } = {
     down: '0',
     left: '90',
     right: '-90',
     up: '180',
   };
-  return directionMap[direction] || direction;
+
+  if (typeof direction === 'number') return direction.toString();
+
+  return directionMap[direction];
```

Comment:
I think it looks better like this:
```ts
return typeof direction === 'number' ? direction.toString() : directionMap[direction];`
```
Also I think we should avoid having conditionals in on line but that's another topic

Reply:
ha ha, I ❤️ one line conditionals 💅🏻 😛 , can't see nothing wrong with them, seems a bit biased as some people think they don't read good and other like me think they read better 💆🏻‍♂️